src/@types/svg.d.ts

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2021 Šimon Brandner <simon.bra.ag@gmail.com>
+Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
This copyright looks a bit incorrect - I think it should just be the foundation?